Packs/MicrosoftSafeLinksStatistics/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Microsoft SafeLinks Statistics",
+    "description": "Enables querying the O365 Safe Links statistics from O365",
```

Comment:
Not exactly a sentence. I think it will look better like this:
"-> Safe Links is a feature in Defender for O365 that provides URL scanning and rewriting of inbound email messages in mail flow, and time-of-click verification of URLs and links in email messages and other location. This pack enables  querying the O365 Safe Links statistics from O365 to filter, block URLs based on your settings. "

Reply:
:) my internal poet is not so this sounds a lot better :)